src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.iOS.cs

```diff
@@ -91,46 +91,46 @@ public async Task CollectionViewContentRespectsMargin()
 			});
 		}
 
-		[Fact("Cells Do Not Leak")]
-		public async Task CellsDoNotLeak()
-		{
-			SetupBuilder();
-
-			var labels = new List<WeakReference>();
-			VerticalCell cell = null;
-
-			{
-				var bindingContext = "foo";
-				var collectionView = new CollectionView
-				{
-					ItemTemplate = new DataTemplate(() =>
-					{
-						var label = new Label();
-						labels.Add(new(label));
-						return label;
-					}),
-				};
-
-				var handler = await CreateHandlerAsync(collectionView);
-
-				await InvokeOnMainThreadAsync(() =>
-				{
-					cell = new VerticalCell(CGRect.Empty);
-					cell.Bind(collectionView.ItemTemplate, bindingContext, collectionView);
-				});
-
-				Assert.NotNull(cell);
-				Assert.NotEmpty(labels);
-			}
-
-			await Task.Yield();
-			GC.Collect();
-			GC.WaitForPendingFinalizers();
-
-			foreach (var reference in labels)
-			{
-				Assert.False(reference.IsAlive, "View should not be alive!");
-			}
-		}
+		//[Fact("Cells Do Not Leak")]
+		//public async Task CellsDoNotLeak()
```

Comment:
Better use just Skip with a comment (the comment will help to understand the status). Also, should we create a related issue where @jonathanpeppers could work?